video/vdu_audio.h

```diff
@@ -12,231 +12,13 @@
 #include <unordered_map>
 #include <fabgl.h>
 
-fabgl::SoundGenerator		SoundGenerator;		// The audio class
-
 #include "agon.h"
 #include "agon_audio.h"
-#include "vdp_protocol.h"
 #include "types.h"
 
-// audio channels and their associated tasks
```

Comment:
code for managing audio channels has been moved into other files.  this file is now solely responsible for handling audio VDU commands